packages/amplify_api/example/lib/main.dart

```diff
@@ -16,6 +16,7 @@
 import 'package:amplify_api/amplify_api.dart';
 import 'package:amplify_api_example/graphql_api_view.dart';
 import 'package:amplify_api_example/rest_api_view.dart';
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
```

Comment:
If the configuration is using an API_KEY would this still be needed

Reply:
Do you know how we might set that up?  I remember that even with setting "NO" for Restrict API Access that I 'd need to add the Auth plugin module still @haverchuck

Reply:
I had to make a similar change to my version of this app to run it, so I'm also interested to see if there is a way around it. Absent some solution there, +1 this change.

Reply:
After discussing with @haverchuck we will be removing the Auth addition changes to the API example app as it's out of scope for the NS upgrade.